Source/Core/Core/IOS/ES/TitleManagement.cpp

```diff
@@ -622,6 +622,51 @@ IPCCommandResult ES::ExportTitleDone(Context& context, const IOCtlVRequest& requ
 {
   return GetDefaultReply(ExportTitleDone(context));
 }
+
+ReturnCode ES::DeleteSharedContent(const std::array<u8, 20>& sha1) const
+{
+  IOS::ES::SharedContentMap map{Common::FromWhichRoot::FROM_SESSION_ROOT};
+  const std::string content_path = map.GetFilenameFromSHA1(sha1);
+  if (content_path == "unk")
```

Comment:
Yeah, I don't like the magic string either. I'm just waiting for std::optional to drop that ;)